src/assets/js/main.js

```diff
@@ -729,3 +729,18 @@ for (var i = 0; i < pathEls.length; i++) {
     autoplay: true
   });
 }
+
+
+var consolestyle = [
+	'background: linear-gradient(to right, #2970ff, #2ED6A1);',
+	'color: #091440',
+	].join(';');
+  
+  console.log(`%c
+  Stakey needs you! for a bug squishin' mission https://docs.decred.org/contributing/overview/
+    ┌ᴗᴗᴗᴗᴗᴗ┐╭  ╮┌ᴗᴗᴗᴗᴗᴗ┐╭  ╮┌ᴗᴗᴗᴗᴗᴗ┐    ┌ᴗᴗᴗᴗᴗᴗ┐╭    ┌ᴗᴗᴗᴗᴗᴗ┐╭  ╮┌ᴗᴗᴗᴗᴗᴗ┐╭  ╮┌ᴗᴗᴗᴗᴗᴗ┐    ┌ᴗᴗᴗᴗᴗᴗ┐╭ 
```

Comment:
The ASCII only works when the console has sufficient width to draw it. The art is distorted otherwise. Perhaps line-wrap could be disabled for the ASCII so it always renders properly regardless of panel size?

We should keep line-wrap on the first line because we want to ensure that all of the relevant text is displayed for every reader.

Reply:
Setting ```'white-space: pre'``` seems to fix this in firefox but not chrome . I dont see this as a huge issue since console is usually used in larger panels  (Also need to define two styles) . Can we drop this ?

Reply:
Or, "your first assignment as a potential contractor is to fix the wrapping"

Reply:
@degeri and I agreed on Matrix to leave this as-is. It provides minimal benefit, it is not trivial to fix, and so its not worth investing more time into.